Conference-Android/app/src/main/AndroidManifest.xml

```diff
@@ -2,53 +2,62 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
BP: should not have to reference parent actvity.  Typically only used for up navigation.  See here:

http://stackoverflow.com/questions/19207762/must-i-specify-the-parent-activity-name-in-the-android-manifest